packages/components/Molecules/Card/CardPromo.vue

```diff
@@ -0,0 +1,273 @@
+<template>
```

Comment:
I think CardPromo is the best compromise name as CardPromotion is already taken and will be deprecated once the Card Consolidation ticket is complete. I'd prefer not to use an interim name like CardPromotionV2 or like.